packages/cozy-client/src/models/document/locales/en.json

```diff
@@ -115,5 +115,47 @@
       "finance": "Finance",
       "activity": "Activities"
     }
+  },
```

Comment:
don't commit .DS_Store 🙏 and try to prevent you having this issue again: https://stackoverflow.com/questions/18393498/gitignore-all-the-ds-store-files-in-every-folder-and-subfolder